galaxychop/core.py

```diff
@@ -364,23 +314,23 @@ def energy(self):
 
         return (Etot_dm, Etot_s, Etot_g)
 
-    def angular_momentum(self, r_corte=None):
+    def angular_momentum(self, r_cut=None):
         """
-        Angular Momentum.
+        Specific Angular Momentum.
 
-        Centers the particles with respect to the one with lower potential,
-        then calculates angular momentum of dark matter, stars and
-        gas particles.
+        Centers the particles with respect to the one with lower specific
```

Comment:
Is not in the most bound particle? ie: the particle with the lower total energy

Reply:
No, it is defined from the particle whose gravitational potential is the minimum.